web/styles/zulip.css

```diff
@@ -82,20 +83,15 @@ body,
     --color-date: hsl(0deg 0% 100% / 75%);
     --color-background-private-message-header: hsl(46deg 15% 20%);
     --color-text-private-message-header: hsl(0deg 0% 100%);
-<<<<<<< HEAD
     --color-background-private-message-content: hsl(46deg 10% 15%);
-||||||| merged common ancestors
-    --color-background-private-message-content: hsl(46deg 25% 14%);
-=======
-    --color-background-private-message-content: hsl(46deg 25% 14%);
     --color-background-stream-message-content: hsl(0deg 0% 14%);
->>>>>>> UI redesign: Change background color.
```

Comment:
Looks like you got some merge conflict markers leaking into intermediate commits.

Reply:
Dang, I noticed that and thought I fixed it but apparently not. Thanks!